salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
You can't just hard code to `latin1`, please test the following:

```python
return json.dumps(data, default=repr, indent=4, encoding=__salt_system_encoding__)
```

Or probably even better:

```python
return json.dumps(data, default=repr, indent=4, encoding=__salt_system_encoding__ if __salt_system_encoding__ != 'ascii' else 'utf-8')
```